project.clj

```diff
@@ -1,12 +1,15 @@
-(defproject lein-ring "0.12.5"
+(defproject dpiliouras/lein-ring "0.12.5"
   :description "Leiningen Ring plugin"
-  :url "https://github.com/weavejester/lein-ring"
+  :url "https://github.com/dpiliouras/lein-ring"
   :scm {:name "git"
-        :url "https://github.com/weavejester/lein-ring"}
+        :url "https://github.com/dpiliouras/lein-ring"}
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/core.unify "0.5.7"]
                  [org.clojure/data.xml "0.0.8"]
                  [leinjacker "0.4.2"
                   :exclusions [org.clojure/core.unify]]]
+  :deploy-repositories [["clojars" {:url "https://clojars.org/repo"
```

Comment:
This can be removed. Pretty much all changes to the `project.clj` file can be reverted.

Reply:
My bad, i will revert all the changes to `project.clj` asap.